src/main/resources/scipooldev.properties

```diff
@@ -1,5 +1,7 @@
 SYNAPSE_OAUTH_CLIENT_ID=100055
-TEAM_TO_ROLE_ARN_MAP=[{"teamId":"273957","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}]
+TEAM_TO_ROLE_ARN_MAP=[{"teamId":"3407239","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, 
```

Comment:
actually, this is a java properties file so it supports comments with `#` character. https://en.wikipedia.org/wiki/.properties

let's add a few comments for this?   Can also be done in a follow on PR.